content/general/events/aws-lambda.textile

```diff
@@ -0,0 +1,163 @@
+---
+title: AWS Lambda Functions
+section: general
+index: 22
+jump_to:
+  Help with:
+    - Fields#fields
+    - Restrictions#restrictions
+    - Example#example
+---
+
+As part of "Reactor Events":/general/events, it is possible to integrate with various systems.
+
+"AWS Lambda Functions":https://aws.amazon.com/lambda provide event-driven serverless compute functions which allow users to easily run code whenever events are sent to it. These can be very useful for integrating into various other AWS systems.
+
+h3(#tutorials). Tutorials & Examples
+
+If you would like to just dive in and see an example of this being implemented, then take a look at our "AWS Lambda tutorial":/tutorials/reactor-event-aws.
+
+h2(#fields). Rule fields
+
+- AWS Region := The region you chose for your "AWS Lambda function":https://docs.aws.amazon.com/general/latest/gr/rande.html#lambda_region
+- Function Name := The name you gave your "AWS Lambda function":https://docs.aws.amazon.com/lambda/latest/dg/getting-started.html
+- AWS Authentication Scheme := the "authentication scheme":https://support.ably.io/support/solutions/articles/3000071908 you wish to use for your function. Either @AWS Credentials@ or @ARN of an assumable role@
+- Qualifier := _optional_ qualifier for your AWS Lambda function
+- "Source":/general/events#sources := Choose which of @Message@ or @Presence@ events on channels should activate this Reactor Event Rule.
+- Channel filter := An optional filter, which allows the Rule to be applied to a restricted set of channels. This can be specified as a regular expression, allowing for swathes of channels to be used
+- Encoding := The encoding to be used by this Rule. This can only be JSON for AWS Lambda Functions
```

Comment:
> The encoding to be used by this Rule. This can only be JSON

why bother even showing & documenting the field if there's only one valid choice?